protocols/luci-proto-wireguard/luasrc/model/cbi/admin_network/proto_wireguard.lua

```diff
@@ -81,10 +81,22 @@ preshared_key = section:taboption(
             "cryptography for post-quantum resistance.")
 )
 preshared_key.password = true
-preshared_key.datatype = "and(base64,rangelength(44, 44))"
+preshared_key.datatype = "and(base64,rangelength(44,44))"
 preshared_key.optional = true
 
 
+fwmark = section:taboption(
+  "advanced",
+  Value,
+  "fwmark",
+  translate("Firewall Mark"),
+  translate("Optional. 32-bit hexadecimal mark for outgoing packets. " ..
+            "Enter value without leading <code>0x</code>.")
```

Comment:
This is silly. `wg(8)` will accept either decimal, without the leading `0x`, or hexadecimal, with the leading `0x`. What you're suggesting here is an awkward hybrid of the two. Instead, just accept to take either, and pass it to `wg(8)`, letting `wg(8)` figure it out.